Editing "primary microplastic particle" and "secondary microplastic particle"

iwilkie opened the pull request:
This PR corresponds to issue #1108. We have added Italian, German and Spanish interpretations and added the terms to the "envoPlastics" subset.

We will add the axioms at a later date.

raissameyer commented on src/envo/envo-edit.owl:
```diff
@@ -27234,14 +27234,28 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_01000944> <http://purl.obolibrar
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0002-2996-719X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0002-8166-0486>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/contributor> <http://purl.obolibrary.org/obo/ENVO_01000945> <https://orcid.org/0000-0003-1072-8081>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasRelatedSynonym> <http://purl.obolibrary.org/obo/ENVO_01000945> "Primäres Mikroplastik"@de)
```
@iwilkie Here and below, I noticed that you added the multilingual interpretations as `has_related_synonym` and not `has_exact_synonym` - was that intentional? At least for the German term I believe it refers to the exact same thing as the English one.

raissameyer reviewed (commented):
Hi @iwilkie - Many thanks for adding the multilingual interpretations!

I just had a brief comment about why you added them as `has_related_synonym` rather than `has_exact_synonym` - was that intentional?

iwilkie commented:
@raissameyer I moved the multilingual interpretations of both terms under `has_exact_synonym`. While I was looking at those terms I noticed that "piece of plastic" and "microplastic particle" were not in the "envoPlastics" subset so I added that annotation to those terms.

raissameyer commented:
@iwilkie great, good that you caught that. 

@pbuttigieg any further comments from your side?
